package.json

```diff
@@ -543,27 +543,48 @@
               "clean",
               "cleanRebuild"
             ],
-            "description": "CMake build command"
+            "description": "%cmake-tools.taskDefinitions.properties.command.description%"
           },
           "targets": {
             "type": "array",
             "items": {
               "type": "string"
-            }
+            },
+            "description": "%cmake-tools.taskDefinitions.properties.targets.description%"
+          },
+          "preset": {
+            "type": "string",
+            "description": "%cmake-tools.taskDefinitions.properties.preset.description%"
           },
           "options": {
             "type": "object",
             "description": "Additional command options",
             "properties": {
               "cwd": {
                 "type": "string",
-                "description": "The current working directory of the executed program or script. If omitted Code's current workspace root is used."
+                "description": "%cmake-tools.taskDefinitions.properties.options.cwd.description%"
+              },
+              "environment": {
+                "type": "object",
+                "items": {
```

Comment:
I don't think we should do environment as "name" "value" explicit pairs.  We only do that for debugConfig to match what VS Code defined for debugging.  It's a very verbose and unfortunate thing.  I think we just want normal string/string key/values in the object.

Reply:
```
	"options": {
		"environment": {
			"myenv1": "myenv1"
		}
	},
```
this is how we define the env in tasks.json. 

for `debugconfig` we define it like this:
`    "cmake.debugConfig": {"environment": [{"name":"myenv1","value": "myenv1"}]}`